src/main/resources/hudson/plugins/favorite/column/FavoriteColumn/column.jelly

```diff
@@ -1,27 +1,16 @@
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
-         xmlns:t="/lib/hudson" xmlns:s="/lib/form">
-  <j:if test="${it.isLoggedIn()}">
-    <j:set var="sortData" value="${it.getSortData(job.getFullName())}"/>
-    <j:set var="starClass" value="${it.getStarClassName(job.getFullName())}"/>
-    <j:set var="sizeClass" value="${it.getSizeClassFromIconSize(subIconSize)}"/>
-    <td data="${sortData}">
-      <a href="#">
-        <span id="fav_${job.getFullName()}" data-fullName="${job.getFullName()}"
-              class="${starClass} ${sizeClass}"/>
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout">
+  <j:set var="sortData" value="${it.getSortData(job.getFullName())}"/>
+  <j:set var="fav" value="${it.isFavorite(job.getFullName())}"/>
+  <j:set var="sizeClass" value="${icons.toNormalizedIconSizeClass(subIconSize)}"/>
+  <td data="${sortData}" class="jenkins-table__cell--tight">
+    <div class="jenkins-table__cell__button-wrapper">
+      <a href="#" class="jenkins-table__button" onclick="toggleFavoriteOnClick(event, this)" data-fullName="${job.getFullName()}">
```

Comment:
This uses inline java code which violates CSP. The onclick behaviour should be attached within the assets.js with `Behaviour.specify(...)`

Reply:
I assume the inline java code you are referencing is `data-fullName="${job.getFullName()}"`. Shouldn't `<?jelly escape-by-default='true'?>` at the top of the jelly file mitigate that? If not can you suggest a remedy.

Moved `onclick` to `Behaviour.specify` ✅